lib/commands/console.js

```diff
@@ -43,20 +50,37 @@ module.exports = {
 
     return setup.then(() => {
       if (!fs.existsSync(dist)) {
-        throw new SilentError('You must first ember your app (console is booted from dist/*)')
+        throw new SilentError('You must first build your ember app (console is booted from dist/*)')
       }
 
       fastbootConsole(dist).then((data) => {
         let { app, response, fastbootConsoleApi } = data;
 
+        if (options.eval || options.evalScript) {
```

Comment:
This isn't the prettiest code but a working example. I will clean it up over the next few days